src/Mod/Path/PathScripts/PostUtilsParse.py

```diff
@@ -94,130 +94,144 @@ def drill_translate(values, outstring, cmd, params):
 
     # get the other parameters
     drill_feedrate = Units.Quantity(params["F"], FreeCAD.Units.Velocity)
-    if cmd == "G83":
+    if cmd in ("G73", "G83"):
         drill_Step = Units.Quantity(params["Q"], FreeCAD.Units.Length)
-        a_bit = (
-            drill_Step * 0.05
-        )  # NIST 3.5.16.4 G83 Cycle:  "current hole bottom, backed off a bit."
+        # NIST 3.5.16.4 G83 Cycle:  "current hole bottom, backed off a bit."
+        a_bit = drill_Step * 0.05
     elif cmd == "G82":
         drill_DwellTime = params["P"]
 
     # wrap this block to ensure machine's values["MOTION_MODE"] is restored
     # in case of error
-    try:
-        if values["MOTION_MODE"] == "G91":
-            trBuff += (
-                linenumber(values) + "G90\n"
-            )  # force absolute coordinates during cycles
-
-        strG0_RETRACT_Z = (
-            "G0 Z"
+    # try:
+    if values["MOTION_MODE"] == "G91":
+        # force absolute coordinates during cycles
+        trBuff += linenumber(values) + "G90\n"
```

Comment:
Python strings are immutable.  So every time you do string concatenation, a new object is created.  String concatenation is considered slow and should be avoided.  Also, with multiple values, the strings get hard to read.
Our minimum python version is now 3.6 so we can use fstrings.  They're fast and very attractive.  Unless there's a reason to not use them, I think we should.  
One reason to not use is in translation but that doesn't apply here.
https://realpython.com/python-f-strings/

Reply:
I have converted all of the strings that I could find to f-strings.

Reply:
The newly pushed version should have all of the changes we talked about in it.
It also passed all of the usual tests, including ./FreeCAD -t 0.